webapp/components/LogInAccordion.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { Typography, Button, Accordion } from '@material-ui/core';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Alert from '@material-ui/lab/Alert';
+
+
+
+import GetAppIcon from '@material-ui/icons/GetApp';
+
+
+import Input from './Input';
+
+import styles from './Form.module.css';
+import { fetchProfile, fetchToken } from 'service/medlemService';
+
+
+type Props = {
+    updateForm: (name: string, mailFrom: string, bankAccount: string, token: string) => void;
+  };
+const LogInAccordion = ({updateForm}: Props): JSX.Element => {
+
+    const [phoneNumber, setPhoneNumber] = useState('+47');
+    const [password, setPassword] = useState('');
+    const [error, setError] = useState('');
+    const [isSuccess, setIsSuccess] = useState(false);
+    const [expanded, setExpanded] = useState(false);
+
+
+    const fetchUserData = async () => {
+        try {
+            const token = await fetchToken(phoneNumber, password);
+            const profileData = await fetchProfile(token);
+            
+            const name = `${profileData.first_name} ${profileData.last_name}` ; 
+            const mailFrom = profileData.email;
+            const accountNumber = profileData.bank_account_number;
+            setError('');
+            setIsSuccess(true);
+            setExpanded(false);
+            updateForm(name, mailFrom, accountNumber, token);
+        }
+        catch (err: any) {
+            if (typeof(err) == 'string') {
+                setError(err)
+            } 
+            else setError("Noe gikk galt.")
+        }
+    }
+    
+return (
+    <Accordion
+        defaultExpanded={false}
+        disabled={isSuccess}
+        expanded={expanded}
+        style={{marginBottom: '2em', paddingLeft: '0.5em', paddingRight: '0.5em'}}
+        onChange={(e, value) => setExpanded(value)}
+        >
+        <AccordionSummary
+            expandIcon={<ExpandMoreIcon />}
+            >
+            <Typography>Auto-fyll med data fra medlem.ntnui.no (valgfritt)</Typography>
+            </AccordionSummary>
+        <Input
+        name="Telefonnummer"
+        value={phoneNumber}
+        updateForm={setPhoneNumber}
+        helperText="Ditt telefonnummer med landskode" />
+        <Input 
+        name="Passord"
+        value={password}
+        updateForm={setPassword}
+        helperText="Ditt passord på medlem.ntnui.no"
+        type="password"/>
+        <Button variant="contained"
+            color="primary"
+            disabled={isSuccess}
+            startIcon={<GetAppIcon/>}
+            style={{ width: '100%', marginTop: '1em', marginBottom: '1em' }}
+            className={styles.fullWidth}
+            onClick={() => fetchUserData()}
+        >
```

Comment:
When writing username and password, clicking enter should click this button ⌨️

Reply:
> Building the Docker image also gives an error after your changes 🤔
> 
> ```
> #23 2.103 info  - Checking validity of types...
> #23 9.780 Failed to compile.
> #23 9.780
> #23 9.781 ./pages/_document.tsx:4:26
> #23 9.781 Type error: Type '() => any' is not a constructor function type.
> #23 9.781
> #23 9.781   2 | import Document, { Html, Head, Main, NextScript } from 'next/document';
> #23 9.781   3 |
> #23 9.781 > 4 | class MyDocument extends Document {
> #23 9.781     |                          ^
> #23 9.781   5 |   // eslint-disable-next-line
> #23 9.781   6 |   static async getInitialProps(ctx: any) {
> #23 9.781   7 |     const initialProps = await Document.getInitialProps(ctx);
> #23 9.829 error Command failed with exit code 1.
> ```

I think this is due to the wrong next version, see #22